tools/getkeys/index.js

```diff
@@ -46,24 +103,6 @@ async function execAsync(command) {
     });
 }
 
-async function setEnv(name, value) {
-    const shell = process.env.SHELL ? process.env.SHELL.split('/').pop() : null;
-    const termProgram = process.env.TERM_PROGRAM;
-    const setString = `export ${name}="${value}"`;
-    switch (shell) {
-        case "bash":
-            const destFile = termProgram === "Apple_Terminal" ? "~/.bash_profile" : "~/.bashrc";
-            return execAsync(`${setString} && echo '${setString}' >> ${destFile}`);
-        case "zsh":
-            return execAsync(`${setString} && echo '${setString}' >> ~/.zshrc`);
-        case "fish":
-            return execAsync(`set -xU '${name}' '${value}'`, { "shell": process.env.SHELL });
```

Comment:
'fish' support was broken so I removed it.

(Node 'exec' spawns '/bin/sh', which means 'set -xU' won't work.  The extra '{ shell }' argument is used.)

Reply:
Yep, that was it.  I've put back 'fish' support.  Thx.